build.gradle.kts

```diff
@@ -214,6 +214,9 @@ dependencies {
     testImplementation("org.seleniumhq.selenium:selenium-support:3.141.59")
     testImplementation("org.seleniumhq.selenium:selenium-chrome-driver:3.141.59")
     testImplementation("org.mockftpserver:MockFtpServer:2.6")
+    testImplementation("org.springframework.ldap:spring-ldap-test")
+    implementation("org.springframework.boot:spring-boot-starter-data-ldap")
```

Comment:
Is this dependency actually required (i.e. `org.springframework.boot:spring-boot-starter-data-ldap`) if so can it be moved up the `spring-ldap-core` dependency.

Reply:
Yep, having it be a required (not a test) dependency was needed to get it to start with the tests correctly. Moved now.